src/reactapp/src/components/paypalExpress/components/PaypalExpress.jsx

```diff
@@ -0,0 +1,56 @@
+import { func, shape, string } from 'prop-types';
+import React, { useEffect } from 'react';
+import RadioInput from '../../common/Form/RadioInput';
+import usePaypalExpress from '../hooks/usePaypalExpress';
+import useCheckoutFormContext from '../../../hook/useCheckoutFormContext';
+
+function PaypalExpress({ method, selected, actions }) {
+  const isSelected = method.code === selected.code;
+  const { authorizeUser } = usePaypalExpress();
+  const { registerPaymentAction } = useCheckoutFormContext();
+
+  useEffect(() => {
+    registerPaymentAction('paypal_express', authorizeUser);
```

Comment:
No need to hard code here. pass `method.code` here

Reply:
Done